src/DateTimePicker.jsx

```diff
@@ -43,6 +48,10 @@ class DateTimePicker extends React.Component {
 			return;
 		}
 
+		if (typeof document === 'undefined') {
```

Comment:
trivial, but you might as well combine this check with the `if (this.props.forceCalendar)`